fb/usage.go

```diff
@@ -50,10 +51,22 @@ type NameID struct {
 	Name string `json:"name"`
 }
 
-func (fbClient FlashbladeClient) Usage() (UsageResponse, error) {
+func FilterFilesystems(vs []FilesystemsItem, regexMatch string) []FilesystemsItem {
+    vsf := make([]FilesystemsItem, 0)
+    for _, v := range vs {
+        matched, _ := regexp.MatchString(regexMatch, v.Name)
+        if matched {
+            vsf = append(vsf, v)
+        }
+    }
+    return vsf
+}
+
+func (fbClient FlashbladeClient) Usage(fsLimitFlag string) (UsageResponse, error) {
```

Comment:
I know there aren't many comments in this project 😳 - but I think a comment on this function explaining that it makes an API call per filesystem, but you can filter it down would be quite useful. What do you think?